<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1537</width>
    <height>973</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>211</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QDoubleSpinBox" name="posX"/>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="xLine">
       <property name="text">
        <string>Posicion X</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QDoubleSpinBox" name="posY"/>
     </item>
     <item row="2" column="0">
      <widget class="QDoubleSpinBox" name="Radio"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="radioLine">
       <property name="text">
        <string>Radio</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QDoubleSpinBox" name="Masa"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="masaLine">
       <property name="text">
        <string>Masa</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QDoubleSpinBox" name="velX"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="velXLine">
       <property name="text">
        <string>Velocidad X</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QDoubleSpinBox" name="velY"/>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="velYLine">
       <property name="text">
        <string>Velocidad Y</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="yLine">
       <property name="text">
        <string>Posicion Y</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>160</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="annadirCuerpo">
       <property name="text">
        <string>Añadir Cuerpo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sistema1">
       <property name="text">
        <string>Sistema 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sistema2">
       <property name="text">
        <string>Sistema 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="eliminarUltimo">
       <property name="text">
        <string>Eliminar Último</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="iniciarSimulacion">
       <property name="text">
        <string>Iniciar Simulación</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="reiniciar">
       <property name="text">
        <string>Reiniciar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>1091</width>
      <height>611</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1537</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSistema_gravitacional">
    <property name="title">
     <string>Sistema Gravitacional</string>
    </property>
   </widget>
   <addaction name="menuSistema_gravitacional"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
